### PR TITLE
Cran prep

### PR DESCRIPTION
**Pull request by graemeblair**

### CODE REVIEW

File: R/modify_level.R [hunk 1]
```diff
@@ -1,3 +1,5 @@
+#' @param by (optional) quoted name of level \code{modify_level} uses to split-modify-combine data.
```
Comment (nfultz) [inline R/modify_level.R]: s/level/variable/ - it need not be an official "fabricator level" - see eg the unit test which uses the sleep data set.

File: R/aaa_fabricate.R [hunk 2]
```diff
@@ -76,6 +76,16 @@
 #'   cities = modify_level(runoff = rnorm(N))
 #' )
 #'
+#' # fabricatr can add variables that are higher-level summaries of lower-level
+#' # variables via a split-modify-combine logic and the \code{by} argument
+#'
+#' multi_level_df <-
```
Comment (nfultz) [inline R/aaa_fabricate.R]: No need to create a variable `multi_level_df` here, just show the usage directly.

Comment (nfultz) [reply R/aaa_fabricate.R]: It may also be worth showing an example of how magic N interacts with by.

Comment (graemeblair) [reply R/aaa_fabricate.R]: Good idea, happy to include if you don't mind sending on such an example.

**Pull request merged**